src/frontend/Sider/Behandling/Inngangsvilkår/Vilkårperioder/EndreVilkårperiode/Begrunnelse.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Textarea } from '@navikt/ds-react';
+
+import { BegrunnelseGrunner, begrunnelseTilTekst } from './utils';
+
+interface Props {
+    begrunnelse?: string;
+    oppdaterBegrunnelse: (nyBegrunnelse: string) => void;
+    delvilkårSomKreverBegrunnelse: BegrunnelseGrunner[];
+    feil?: string;
+}
+
+const Liste = styled.ul`
+    margin: 0.25rem 0;
+`;
+
+const Begrunnelse: React.FC<Props> = ({
+    begrunnelse,
+    oppdaterBegrunnelse,
+    delvilkårSomKreverBegrunnelse,
+    feil,
+}) => {
+    const begrunnelseSuffix =
+        delvilkårSomKreverBegrunnelse.length > 0 ? '(obligatorisk)' : '(valgfri)';
+
+    return (
+        <Textarea
+            label={`Begrunnelse ${begrunnelseSuffix}`}
+            description={
+                delvilkårSomKreverBegrunnelse.length > 0 && (
+                    <>
+                        Du må begrunne vurderingen av:
+                        <Liste>
```

Comment:
Kanskje kan bruke https://aksel.nav.no/komponenter/core/list her?

Reply:
Må overskrive stylingen en del for å få den til å funke
<img width="521" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/8aa6254b-7cca-4ebd-aacf-3f064ea88be1">

Reply:
Selv om man styler den uten padding vil den få en annen font enn det som kommer default i description. Så teksten over punktene matcher ikke og `title` kan ikke brukes fordi det ble større enn labelen på feltet. 

<img width="807" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/2c991d96-7601-4615-8c87-8904b7810345">
<img width="793" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/1b997fe9-ff19-4922-9529-263fd865d650">